docs/topics/runners/dokka-gradle.md

```diff
@@ -531,188 +452,236 @@ typically have the following structure:
 </tab>
 </tabs>
 
-In this case, there are multiple ways of applying and configuring Dokka.
+Single and multi-module documentation share the same 
+[configuration model using the top-level `dokka {}` DSL](#single-project-configuration).
+In multi-module projects, 
+you configure Dokka in the root project and can optionally share settings across subprojects or modules.
 
-#### Subproject configuration
+To share Dokka configuration across modules, you can use either:
 
-To configure subprojects in a multi-project build, you need to configure [`Partial`](#partial-tasks) tasks.
+* [Direct configuration in multi-module projects without convention plugins](#direct-configuration-in-multi-module-projects-without-convention-plugins)
+* [Convention plugins](#multi-module-projects-with-convention-plugins)
 
-You can configure all subprojects at the same time in the root `build.gradle.kts` or `build.gradle` file,
-using Gradle's `allprojects {}` or `subprojects {}` configuration blocks:
+After sharing Dokka configuration, you can aggregate the documentation from multiple modules into a single output. 
+For more information, see
+[Aggregate documentation output in multi-module projects](#aggregate-documentation-output-in-multi-module-projects).
 
-<tabs group="build-script">
-<tab title="Kotlin" group-key="kotlin">
-
-In the root `./build.gradle.kts`:
-
-```kotlin
-import org.jetbrains.dokka.gradle.DokkaTaskPartial
+> For a multi-module project example, see the [Dokka GitHub repository](https://github.com/Kotlin/dokka/tree/master/examples/gradle-v2/multimodule-example).
+>
+{style="tip"}
 
-plugins {
-    id("org.jetbrains.dokka") version "%dokkaVersion%"
-}
+#### Direct configuration in multi-module projects without convention plugins
 
-subprojects {
-    apply(plugin = "org.jetbrains.dokka")
+If your project doesn't use convention plugins, you can share Dokka configurations by directly configuring each module.
+This involves manually setting up the shared configuration in each module's `build.gradle.kts` file. 
+While this approach is less centralized,
+it avoids the need for additional setups like convention plugins.
 
-    // configure only the HTML task
-    tasks.dokkaHtmlPartial {
-        outputDirectory.set(layout.buildDirectory.dir("docs/partial"))
-    }
+Follow the next steps to configure your multi-module project without convention plugins.
```

Comment:
I don't get this part. It should be the next steps without convention plugins, but then we start describing how to create a convention plugin.

Reply:
"Without" means that require one, that's why we have the instructions to create one. I've updated it, hopefully it is clearer.

Reply:
@adam-enko I remember writing these instructions with content you provided. Any idea about how to better differenciate `Direct configuration in multi-project builds requiring convention plugins﻿` vs `Multi-project builds with convention plugins﻿` sections?